spec/spec.yaml

```diff
@@ -8,7 +8,17 @@ python3:
   entrypoint: 'python3'
   filename:   'main.py'
 
+rust:
+  image:      'rust:alpine'
+  entrypoint: 'rustc main.rs && ./main'
+  filename:
```

Comment:
Please change `etrypoint` to `cmd` to overwrite the command. Also, please add `main.rs` as file name.

The file name must be provided so that the File Provider is able to write the source code file and mount it into the execution container.  
Therefore, `cmd` must be overwritten in this case, otherwise, the file name would be used as `cmd` and the command to start the container would be like `rust main.rs && ./main main.rs`. ^^